jax_galsim/sum.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+import jax.numpy as jnp
+
+from jax_galsim.gsparams import GSParams
+from jax_galsim.gsobject import GSObject
+
+import galsim as _galsim
+from jax._src.numpy.util import _wraps
+from jax.tree_util import register_pytree_node_class
+
+
+@_wraps(
+    _galsim.Add, lax_description="Does not support `ChromaticObject` at this point."
+)
+def Add(*args, **kwargs):
+    return Sum(*args, **kwargs)
+
+
+@_wraps(
+    _galsim.Sum, lax_description="Does not support `ChromaticObject` at this point."
```

Comment:
where would this description be surfaced?

Reply:
it would appear in the documentation like for instance in this example from jax,scipy https://jax.readthedocs.io/en/latest/_autosummary/jax.scipy.ndimage.map_coordinates.html
![image](https://user-images.githubusercontent.com/861591/172426253-6d4b0ffc-f687-430d-8312-bbed8f208c1d.png)

Reply:
thanks!